### ISSUE TITLE
Regression: striping of font family, font-size, replacing by incorrect values

### ISSUE THREAD

**Issue by techdragon** [mjmlio/mjml#132]
I've had to resort to forced style values on inner tags in order to fix the stripping of font values for instance on mj-text here: 
```html
<mj-text padding="0 5px" font-size="20" font-family="-apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Roboto&quot;, &quot;Oxygen&quot;, &quot;Ubuntu&quot;, &quot;Cantarell&quot;, &quot;Fira Sans&quot;, &quot;Droid Sans&quot;, &quot;Helvetica Neue&quot;, sans-serif">
<a style="color:#039be5;font-size:20px; font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Roboto&quot;, &quot;Oxygen&quot;, &quot;Ubuntu&quot;, &quot;Cantarell&quot;, &quot;Fira Sans&quot;, &quot;Droid Sans&quot;, &quot;Helvetica Neue&quot;, sans-serif;" href='{{link_url}}'>{{heading_text}}</a>
</mj-text>
```

Turns into

```html
<div style="cursor:auto;color:#000000;font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:13px;line-height:22px;">
<a style="color:#039be5;font-size:20px; font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Roboto&quot;, &quot;Oxygen&quot;, &quot;Ubuntu&quot;, &quot;Cantarell&quot;, &quot;Fira Sans&quot;, &quot;Droid Sans&quot;, &quot;Helvetica Neue&quot;, sans-serif;" href='{{link_url}}'>{{heading_text}}</a>
</div>
```

I didn't specify Arial or Helvetica so why are they being inserted? 
If this isn't a bug how do I fix it, and if it is, then its one that seriously affects the usefulness of mjml (1.3.0-beta4)
Right now I'm left with cheap hacks jamming styles in where I can to override the 'forced default' and significant editing off the generated template by hand, which further defeats the purpose of MJML.

Edit: I had the incorrect beta version. Dont know where I got 8 from, the latest is 4.

**Comment by arnaudbreton** [comment]
Hey @techdragon, sorry to read that you're experiencing an issue here. We'll have a look shortly and come back to you very shortly with a solution.

Thanks for sharing.

**Comment by iRyusa** [comment]
Hi @techdragon 

Will be fix in the next release ! Thanks for reporting

**Comment by iRyusa** [comment]
Fix in `1.3` :+1:
